cron_scripts/inactive_idle_accounts.pl

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+use Getopt::Long;
+use Data::Dumper;
+use JSON;
+use File::Slurp;
+use POSIX qw(strftime);
+use Pod::Usage qw(pod2usage);
+
+=head1 NAME
+
+inactive_idle_accounts.pl - to make idle accounts to inactive status
+
+=head1 SYNOPSIS
+
+  inactive_idle_accounts.pl [-help -debug] -last_login_days <#no. of days> -new_status <#zimbraAccountStatus>
+
+=head1 DESCRIPTION
+
+This script will go through Zimbra's Internal LDAP and look for any account that has a last login greater than the specified days, 
+the script will change their account status to new status that passed to the script
```

Comment:
suggestion: change "the script will change their account status to new status that passed to the script"
to something like (period ends previous sentence) "Matching accounts will have their account status updated to the value provided by the -new_status argument."

Reply:
Done